### ISSUE TITLE
processes array_interpolate_linear does not seem to exist anymore

### ISSUE THREAD

**Issue by hurielreichel** [Open-EO/openeo-r-client#137]
All of of a sudden, on openeo 1.2.1 (same happens for 1.2.2), process array_interpolate_linear does not seem to exist any more. 

processes = list_processes()
login()
p = processes()
w = 6.09
s = 46.15
e = 6.11
n = 46.17

date1 = "2019-01-01"
date2 = "2019-12-31"

datacube = p$load_collection(
    id = "TERRASCOPE_S5P_L3_NO2_TD_V1",
    spatial_extent = list(west = w, south = s, east = e, north = n),
    temporal_extent=c(date1, date2),
    bands=c("NO2")
  )

interpolate = function(data,context) {
  return(p$array_interpolate_linear(data = data))
}

datacube = p$apply_dimension(process = interpolate,
               data = datacube, dimension = "t"
               )
`
When trying to run the above code, I receive a : "Error in (function (data, context) : attempt to apply non-function"

And in RStudio, when I try doing `print(p$array_interpolate_linear`, it's NULL. 

Full Error log : 
6.
(function (data, context) 
{
return(p$array_interpolate_linear(data = data))
})(data = <environment>, context = <environment>)
5.
do.call(value, args = process_graph_parameter)
4.
arguments[[param_name]]$setValue(call_arg) at <text>#15
3.
FUN(X[[i]], ...)
2.
lapply(names(this_arguments), function(param_name, arguments) {
call_arg = this_arguments[[param_name]]
arguments[[param_name]]$setProcess(node)
arguments[[param_name]]$setValue(call_arg) ...
1.
p$apply_dimension(process = interpolate, data = datacube, dimension = "t")`
`

PS : This was a working a function since some hours ago... tested many times.

**Comment by flahn** [comment]
The function in the process collections are created from the processes available at the back-end. I checked also `p$array_interpolate_linear` and it is NULL, also `describe_process("array_interpolate_linear")` returns a message that the process does not exist. Checked also with the web editor and it is the same. This is no client issue then and should be directed towards the back-end provider. Maybe @soxofaan or @jdries can help and clarify this.
